native_mate/function_template.h

```diff
@@ -21,6 +21,9 @@ namespace internal {
 
 struct Destroyable {
   static void Destroy(Arguments* args) {
+    if (IsDestroyed(args))
+      return;
+
```

Comment:
What issue is this addressing? In what case is it valid for this to be called when IsDestroyed() is true (or for us to call GetWrapper() when wrapper_ is empty)?

Reply:
Ah now I see :) Looking at electron/electron#10527 and possibly electron/electron#10433 now...

Reply:
Ah the `IsDestroyed` thing is probably unrelated to those two bugs. I was reading through related Muon changes and noticed this in https://github.com/brave/muon/pull/323/files
Seemed like a reasonable thing to do so figured I'd propose it as well